src/main/java/com/google/sps/data/DBReferences.java

```diff
@@ -0,0 +1,18 @@
+package com.google.meltingpot;
```

Comment:
Please keeps the sps bit - com.google.sps.meltingpot. I think we are using it to indicate it is a STEP project. Hari's PR already have this change - you may want to try merge it.

Reply:
Done!